analyzers/tests/SonarAnalyzer.UnitTest/TestCases/BooleanLiteralUnnecessary.Fixed.cs

```diff
@@ -20,6 +21,7 @@ public BooleanLiteralUnnecessary(bool a, bool b, bool? c, Item item)
             z = false;      // Fixed
             z = true;     // Fixed
             z = false;      // Fixed
+            z = (false);    // Fixed
```

Comment:
This looks like it would also benefit from `WithAnnotation(Simplifier.Annotation)`. Consider adding it, if you still find time for it.

Reply:
I simplified other fixes actually, the problem with this one is that the parentheses are outside the issue/BinaryExpression, thus it's quite tricky to remove that. But still not a big issue